dev_utils/certfixer/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.16
+
+RUN apk add openssl
+
+RUN mkdir -p /certfixer
+
+ADD ./make_certs.sh /certfixer/make_certs.sh
+ADD ./ssl.cnf /certfixer/ssl.cnf
+
+WORKDIR /certfixer
+
+CMD ["/bin/sh", "make_certs.sh"]
```

Comment:
This is not needed, you can use the `neicnordic/sda-helm-tests-support` container that already exists.